glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -556,6 +557,25 @@ open class GleanInternalAPI internal constructor () {
     internal fun testHasPingType(pingName: String): Boolean {
         return LibGleanFFI.INSTANCE.glean_test_has_ping_type(handle, pingName).toBoolean()
     }
+
+    @VisibleForTesting
```

Comment:
Why is this visible for testing? Doesn't seem to be directly tested anyway.

Reply:
Because there is a test associated with this function and variable that I hadn't copied over from AC yet.